presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,68 @@
+  than the configured amount, data recovery and reassignment will not run.
+* Add configuration property ``raptor.startup-grace-period`` to control the delay of the initial bucket balancer run after the coordinator startup.
+
+Hive Connector Changes
+----------------------
+
+* Add ``hive.ignore-table-bucketing`` configuration property and ``ignore_table_bucketing`` session property.
+  When set to true, these properties enable reading from partitions whose bucketing scheme does not match the table bucketing scheme.
+
+Verifier Changes
+----------------
+* Fix an issue where the determinism check for ``INSERT`` queries were not run when there was a result mismatch.
+* Fix an issue where the checksum query was treated as the main query, and the main query was treated as part of the setup queries,
+  for ``INSERT`` query verification.
+* Fix query timeout enforcement by replacing local timer with session property ``query_max_execution_time``.
+* Improve result comparison for floating point columns by using relative errors.
+  Replace configuration property ``expected-double-precision`` with ``relative-error-margin``.
+* Improve result comparison for orderable array columns by applying :func:`array_sort` before :func:`checksum`.
+* Improve result comparison for ``SELECT`` queries by rewriting ``SELECT`` queries as ``CREATE_TABLE_AS_SELECT`` and using checksum queries to verify the results.
+  This eliminates the row count limit for ``SELECT`` queries.
+* Improve Verifier interfaces to allow easy extension by community users.
+* Reuse initial control query results for the determinism check. This reduces the maximum number of control query runs and eliminates the test query reruns.
+* Add support for retrying transient query failures, using configuration properties ``presto.max-attempts``, ``presto.min-backoff-delay``,
+  ``presto.max-backoff-delay``, ``presto.backoff-scale-factor``. Similar configurations prefixed with ``cluster-connection`` instead of ``presto``
+  are introduced for retrying transient network failures when communicating with the coordinator. Intermediate failures are recorded and emitted in the output.
+* Add support for automatically resolving certain kinds of failures including exceeding the global memory limit and exceeding time limit.
+* Add configuration properties ``metadata-timeout`` and ``checksum-timeout`` to set the timeouts for metadata queries
+  (i.e., describe queries that reads table schema) and checksum queries.
+* Add ``source-query.table-name`` configuration property to specify the name of the MySQL table from which the verifier queries will loaded.
+* Add configuration property ``human-readable.log-file`` to allow human-readable verification results to be logged into the specified file instead of to ``stdout``.
+* Rename configuration properties ``query-database`` to ``source-query.database``, ``suites`` to ``source-query.suites``,
```

Comment:
Just curious, will the old config names continue working and just emit a warning, OR will it fail to start/work?

Reply:
They'll fail to work.